Assets/MixedRealityToolkit/Providers/Hands/HandJointUtils.cs

```diff
@@ -54,7 +54,7 @@ public static IMixedRealityHand FindHand(Handedness handedness)
                 var hand = detectedController as T;
                 if (hand != null)
                 {
-                    if (detectedController.ControllerHandedness == handedness)
+                    if ((detectedController.ControllerHandedness & handedness) != 0)
```

Comment:
Please also update the remarks on FindHand above (The given handeness should be either Handedness.Left or Handedness.Right.) - I would just delete the remarks. 

I don't have the ability to suggest the change because that line is in the unchanged section.